src/ontology/uberon-edit.obo

```diff
@@ -224180,6 +224180,16 @@ is_a: UBERON:0000029 ! lymph node
 relationship: dc-contributor http://orcid.org/0000-0002-1773-2692
 property_value: dcterms-date 2022-05-26T11:59:51Z xsd:dateTime
 
+[Term]
+id: UBERON:8480000
+name: iliac vein smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of an iliac vein." []
+synonym: "muscle of iliac vein" BROAD []
+is_a: UBERON:0004696 ! venous system smooth muscle
```

Comment:
I would instead consider equivalent class of:
'smooth muscle tissue' and ('part of' some 'iliac vein')

The reasoner should classify it a venous system smooth muscle 
It's usually good practice to let reasoners do the work to ensure that you dont miss out classifications, fall into some unexpected hidden GCI, etc. furthermore axioms should reflect the definition in logical axioms as far as possible.